client/caps.go

```diff
@@ -45,20 +45,21 @@ type Capability struct {
 
 // Capabilities returns the capabilities currently available for snaps to consume.
 func (client *Client) Capabilities() (map[string]Capability, error) {
+	const errPrefix = "cannot obtain capabilities"
 	var rsp response
 	if err := client.do("GET", "/1.0/capabilities", nil, &rsp); err != nil {
 		return nil, err
```

Comment:
Yes, it was matched with (equally wrong) code in the CLI interface. Fixed both, now when this fails you'll get this error message:

(amd64)ubuntu@localhost:~$ sudo ./snap list-caps
error: cannot obtain capabilities: failed to communicate with server: Get http://localhost/1.0/capabilities: dial unix /run/snapd.socket: connect: connection refused